src/transformers/models/esm/modeling_esm.py

```diff
@@ -377,7 +377,7 @@ def forward(
         if head_mask is not None:
             attention_probs = attention_probs * head_mask
 
-        context_layer = torch.matmul(attention_probs, value_layer)
+        context_layer = torch.matmul(attention_probs.to(value_layer.dtype), value_layer)
```

Comment:
This was needed to perform correctly inference otherwise you get dtype mismatch

Reply:
what do we get if we don't do this fix ?

Reply:
You get a dtype mismatch :/